k8sfv/k8sfv.go

```diff
@@ -167,7 +167,8 @@ func initialize(k8sServerEndpoint string) (clientset *kubernetes.Clientset) {
 			return
 		}
 
-		ctx, _ := context.WithTimeout(context.Background(), 10*time.Second)
+		ctx, cancelfun := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
Nit: use camelCase for locals `cancelFunc`

Reply:
Fixed